### PR TITLE
add new document for v2 quickstart

### PR DESCRIPTION
**Pull request by KPostOffice**
## Description
closes: https://github.com/project-codeflare/codeflare-operator/issues/254

## How Has This Been Tested?
I walked through the steps and made sure that all the pods that should come up did.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: Quick-Start-ODH-V2.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+make all-in-one
+```
+
+> Note : Execute ```make help``` to list additional available operations.
+
+## Prerequisites
+
+### Resources
+
+In addition to the resources required by the odh-core deployment, you will need the following to deploy the Distributed Workloads stack infrastructure pods:
```
Comment (sutaakar) [inline Quick-Start-ODH-V2.md:24]: `In addition to the resources required by the odh-core deployment` - this sentence refers to the KfDef, may be good to reword it to something like `In addition to the resources required by default ODH deployments`.

File: Quick-Start-ODH-V2.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+1. Create the opendatahub namespace with the following command:
+
+    ```bash
+    oc create ns opendatahub
+    ```
+
+1. Create a datascience cluster with CodeFlare and Ray enabled:
+
+    ```bash
+    oc apply -f https://raw.githubusercontent.com/opendatahub-io/distributed-workloads/main/codeflare-dsc.yaml -n opendatahub
```
Comment (sutaakar) [inline Quick-Start-ODH-V2.md:70]:
```suggestion
    oc apply -f https://raw.githubusercontent.com/opendatahub-io/distributed-workloads/main/codeflare-dsc.yaml
```
DSC CR is cluster wide, there is no need to provide namespace.

File: Quick-Start-ODH-V2.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    ```
+
+Applying the above kfdef will result in the following objects being added to your cluster:
```
Comment (sutaakar) [inline Quick-Start-ODH-V2.md:73]:
```suggestion
Applying the above DataScienceCluster will result in the following objects being added to your cluster (apart from default ODH objects):
```

Comment (KPostOffice) [reply Quick-Start-ODH-V2.md]: I fixed the kfdef reference, but I wasn't sure about the statement you put in parenthesis. It felt awkward to me here. Do you think it is necessary?

Comment (sutaakar) [reply Quick-Start-ODH-V2.md]: no, it is just a suggestion, main point was the kfdef reference.

**Review by sutaakar — COMMENTED** [review]
/lgtm

File: Quick-Start-ODH-V2.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    oc get appwrappers -A
+    ```
+     If any are left, you'd want to delete them
+
+2. Remove the notebook and notebook pvc:
+   ```bash
+   oc delete notebook jupyter-nb-kube-3aadmin -n opendatahub
+   oc delete pvc jupyterhub-nb-kube-3aadmin-pvc -n opendatahub
+   ```
+3. Remove the codeflare-stack kfdef: (Removes MCAD, InstaScale, KubeRay and the Notebook image)
```
Comment (sutaakar) [inline Quick-Start-ODH-V2.md:120]:
```suggestion
3. Remove the codeflare-stack: (Removes MCAD, InstaScale, KubeRay and the Notebook image)
```

**Comment by sutaakar** [comment]
@anishasthana @astefanutti can this be merged? I would like to leverage the DSC CR for OpenShift CI

**Review by anishasthana — APPROVED** [review]
/lgtm
